services/brig/src/Brig/API/Types.hs

```diff
@@ -106,6 +106,16 @@ data PasswordResetError
     | InvalidPasswordResetCode
     | ResetPasswordMustDiffer
 
+data LegalHoldLoginError
+    = LegalHoldLoginNoBindingTeam
+    | LegalHoldLoginLegalHoldNotEnabled
+    -- same as LoginError
```

Comment:
what about this then?:

```haskell
data LegalHoldLoginError
    = LegalHoldLoginNoBindingTeam
    | LegalHoldLoginLegalHoldNotEnabled 
    | LegalHoldLoginError LoginError
```

should it ever change, it'll be easy to switch back to your implementation and then mutate it at will.  but if it doesn't change it's much easier to read.